ssdeep/ssdeep-lib/ac-fix/Makefile.am

```diff
@@ -0,0 +1,76 @@
+
```

Comment:
are those files really necessary? cant you just download the release tarball or clone the git repo?